cli.js

```diff
@@ -209,11 +207,16 @@ function notChoked(result, wire) {
   return result + (wire.peerChoking ? 0 : 1)
 }
 
-function getInfo (infile, cb) {
+function getSource (infile, cb) {
+  if (/^magnet:/.test(infile)) return cb(infile)
   var instream = !infile || infile === '-'
     ? process.stdin
     : fs.createReadStream(infile)
-  instream.pipe(concat(function (body) {
+  instream.pipe(concat(cb))
+}
+
+function getInfo (infile, cb) {
+  getSource(function (body) {
```

Comment:
this line was causing `torrent ls` to crash, the fix was to change it to `getSource(infile`